Stripe/STPAnalyticsClient.m

```diff
@@ -410,7 +410,7 @@ + (NSMutableDictionary *)commonPayload {
 
 + (NSDictionary *)serializeConfiguration:(STPPaymentConfiguration *)configuration {
     NSMutableDictionary *dictionary = [NSMutableDictionary dictionary];
-    dictionary[@"publishable_key"] = configuration.publishableKey ?: @"unknown";
+    dictionary[@"publishable_key"] = [STPAPIClient sharedClient].publishableKey ?: @"unknown";
```

Comment:
I think this should be [Stripe defulatPublishableKey]

Reply:
As it stands, STPAPIClient sharedClient is still more likely to be accurate of the two, since if they differ, it means the user set STPAPIClient.sharedClient.publishableKey.

e.g. You could never call Stripe.setDefaultPublishableKey and only set STPAPIClient.sharedClient.publishableKey.

Reply:
mm, I have the opposite understanding. You should _always_ call `stripe.setDefaultPublishableKey` and only set `STPAPIClient.sharedClient.publishableKey` for very specific use cases wher eyou know what you're doing. Otherwise what's the point of Stripe.defaultPublishableKey?